desktop_version/src/Maths.h

```diff
@@ -6,6 +6,7 @@
 
 //// This header holds Maths functions that emulate the functionality of flash's
 
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))
```

Comment:
The game already uses `std::min()` in various places.

Reply:
Hm my build failed when I tried

Reply:
It shouldn't fail, because otherwise the build would have already failed on these usages of `std::min()` and `std::max()`.
![std::min() and std::max() in VVVVVV](https://user-images.githubusercontent.com/59748578/74212324-802cd700-4c48-11ea-9011-97213e64fa38.png)
I think your build failed because you failed to cast the `.size()`s to `int`s (see comment below) when plugging them in to `std::min()`, so the compiler complained.

Reply:
Ah it's because `.size()`. Returns a `size_t` and the first argument is an `int`. It's midnight here :p

Reply:
Ha that's day 2 of you typing something at the same time as me